ITRStartTask/StoryDetail/Presenter/StoryPresenter.swift

```diff
@@ -31,7 +31,7 @@ class StoryPresenter: StoryPresenterProtocol {
             let ids = ids.count > 5 ? Array(ids[0..<5]) : ids
```

Comment:
Magic numbers should be removed from code...

Reply:
fixed